components/concept/engine/src/main/java/mozilla/components/concept/engine/content/blocking/TrackingProtectionExceptionStorage.kt

```diff
@@ -48,7 +48,7 @@ interface TrackingProtectionExceptionStorage {
     /**
      * Removes all domains from the exception list.
      */
-    fun removeAll()
+    fun removeAll(activeSessions: List<EngineSession?>)
```

Comment:
Adding a new parameter will cause a breaking change on the API. I believe we are the only consumers but to be sure, you can add a default a value  `fun removeAll(activeSessions: List<EngineSession?> = null)`. also we need to add some docs about this new parameter :)